### PR TITLE
basic configurations for spring is done

### PR DESCRIPTION
**Pull request by egolesor**
Basic configured the spring is done. now you we can use the features of spring to create our web application

### CODE REVIEW

File: WebClient/pom.xml [hunk 1]
```diff
@@ -7,15 +7,83 @@
...
+        <webapp-runner.version>8.0.33.1</webapp-runner.version>
+        <servlet-api.version>2.5</servlet-api.version>
+        <spring-core.version>4.3.2.RELEASE</spring-core.version>
+        <spring-web.version>4.3.2.RELEASE</spring-web.version>
+
+
+    </properties>
 
     <dependencies>
+        <!-- https://mvnrepository.com/artifact/javax.servlet/servlet-api -->
```
Comment (LukaszCzubernat) [inline WebClient/pom.xml]: Delete these comments about maven repository

File: WebClient/pom.xml [hunk 2]
```diff
@@ -7,15 +7,83 @@
...
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>${servlet-api.version}</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.springframework/spring-core -->
```
Comment (LukaszCzubernat) [inline WebClient/pom.xml]: Same here

File: WebClient/pom.xml [hunk 3]
```diff
@@ -7,15 +7,83 @@
...
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>${spring-core.version}</version>
+        </dependency>
+
+
+        <!-- https://mvnrepository.com/artifact/org.springframework/spring-web -->
```
Comment (LukaszCzubernat) [inline WebClient/pom.xml]: And again

Comment (egolesor) [reply WebClient/pom.xml]: Done

File: WebClient/pom.xml (hunk 2 above)
Comment (egolesor) [reply WebClient/pom.xml]: Done

File: WebClient/pom.xml (hunk 1 above)
Comment (egolesor) [reply WebClient/pom.xml]: Done

File: WebClient/pom.xml [hunk 4]
```diff
@@ -7,15 +7,83 @@
...
+        To debug you have to run the jar file with
+                -agentlib:jdwp=transport=dt_socket,server=y,address=8000,suspend=n
+    -->
+    <build>
+        <plugins>
+            <!-- -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.3</version>
```
Comment (caroo7) [inline WebClient/pom.xml]: Maybe move this 'version' to properties section, like in dependencies.

Comment (egolesor) [reply WebClient/pom.xml]: Done

**Pull request merged**